docs/docs/bin/create-image.js

```diff
@@ -0,0 +1,101 @@
+// eslint-disable-line
+import path from 'path';
+import fs from 'fs';
+import { fileURLToPath } from 'url';
+import pack from 'maplibre-gl/package.json' assert { type: 'json' };
+import puppeteer from 'puppeteer';
```

Comment:
We need to remove this dependency, this tool is basically deprecated.
We also should consider migrating this file to typescript.